Dribbblish-dynamic/dribbblish.js

```diff
@@ -0,0 +1,300 @@
+// Hide popover message
```

Comment:
Rename file to `dribblish-dynamic.js` to avoid ambiguity

Reply:
This means that you will need to update the instructions in the README too

Reply:
renamed and updated README